src/components/FileRepo/FileRepo.js

```diff
@@ -50,7 +53,33 @@ const customTableTypes = {
   ),
 };
 
-const FileRepo = compose(injectState, withTheme, withApi)(
+const generateHeaderContent = (props, sqon) => {
+  props.disabled = false; //props.selectedTableRows == 0; @NOTE Keep functionality for next iteration of the ticket
+  props.sqon = sqon;
```

Comment:
These mutations are very dangerous as other things may accidentally read these values and it'd be very hard to trace it back here.
I'd suggest making `generateHeaderContent` a `TableHeaderContent` component (or something along that line) and passing `sqon` and `false` in as part of the component's props?